.kokoro/presubmit/node10/conformance-test.cfg

```diff
@@ -0,0 +1,4 @@
+env_vars: {
```

Comment:
If you look at the internal job configs, you see that we only have node12 configured. This file should be here: https://github.com/googleapis/nodejs-storage/tree/main/.kokoro/presubmit/node12